nautobot/extras/migrations/0060_role_and_alter_status.py

```diff
@@ -37,12 +37,6 @@ class Migration(migrations.Migration):
                     models.JSONField(blank=True, default=dict, encoder=django.core.serializers.json.DjangoJSONEncoder),
                 ),
                 ("name", models.CharField(max_length=100, unique=True)),
-                (
-                    "slug",
-                    nautobot.core.models.fields.AutoSlugField(
-                        blank=True, max_length=100, populate_from="name", unique=True
-                    ),
-                ),
```

Comment:
Is it a good practice, to alter a migration file not created by this PR?

Reply:
Since this migration is specific to `next`, which hasn't shipped a non-alpha release yet, it's permissible. We generally wouldn't want to do this in `develop` to affect existing released migrations without extensive testing for compatibility.